models/phold/application.c

```diff
@@ -13,21 +13,25 @@ void ProcessEvent(lp_id_t me, simtime_t now, unsigned event_type,
 	(void)content;
 	(void)size;
 	switch (event_type) {
-	case INIT:
+	case LP_INIT:
 		state = malloc(sizeof(unsigned));
 		*state = 0;
 		SetState(state);
 		break;
-	case DEINIT:
+	case LP_FINI:
 		free(state);
 		return;
+	case MODEL_INIT:
+	case MODEL_FINI:
+		return;
 	default:
 		++*state;
 		break;
 	}
 
 	lp_id_t dest = Random() * n_lps;
-	ScheduleNewEvent(dest, now + fabs(Normal() * 5.0) + 20.0, 1, NULL, 0);
+	ScheduleNewEvent(dest, now + fabs(Normal() * 5.) + 20., MODEL_FINI + 1,
```

Comment:
See my other comment to rootsim_event.

Also, I would like to avoid hacks like MODEL_FINI + 1, which make the understanding of the example models too bound to the internals of the runtime.

Reply:
This will be changed for sure once you take a decision on my previous comment

Reply:
Done! :smile: